build/updatehistorictzdb.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+set -e
+
+rm -rf tmp
```

Comment:
I'm expecting to run it from the directory in which it resides - so presumably `cd $(dirname $0)` with no `..`? Or I could just stick to knowing that it'll only be me running it, probably :)